.kitchen.yml

```diff
@@ -56,3 +59,19 @@ suites:
   attributes:
     varnish:
       version: '4.0'
+      log_daemon: false
+      
+- name: distro_libraries
```

Comment:
'distro' means it is installed from the Yum or Apt repositories. Vendor means it is installed from vendor-cache.org repositories. There are no source installs.